ofrak_core/test_ofrak/unit/test_ofrak_server.py

```diff
@@ -1222,11 +1225,21 @@ async def test_get_project_by_id(ofrak_client: TestClient):
     resp = await ofrak_client.get("/get_project_by_id", params={"id": id})
     assert resp.status == 200
     body = await resp.json()
-    assert list(body.keys()) == ["name", "project_id", "session_id", "scripts", "binaries"]
-    shutil.rmtree("/tmp/test-ofrak-projects")
+    assert list(body.keys()) == [
+        "name",
+        "project_id",
+        "session_id",
+        "resource_ids",
+        "scripts",
+        "binaries",
+    ]
+    if os.path.exists("/tmp/test-ofrak-projects"):
+        shutil.rmtree("/tmp/test-ofrak-projects")
 
 
 async def test_get_all_projects(ofrak_client: TestClient):
+    if os.path.exists("/tmp/test-ofrak-projects"):
```

Comment:
With this duplicated everywhere it might have made sense as a context manager or fixture with a setup/cleanup to do this. But that's not blocking this PR. I'll just throw that idea out there, you can decide if you like it and want to implement it.

Reply:
yeah i agree, fixture is appropriate here, it happened sort of organically so i didn't bother, but will update in an additional PR later.